src/SFA.DAS.RequestApprenticeTraining.Api/Controllers/EmployerRequestController.cs

```diff
@@ -198,10 +199,12 @@ public async Task<IActionResult> SubmitProviderResponse(long ukprn, SubmitProvid
             try
             {
                 var response = await _mediator.Send(new SubmitProviderResponseCommand
-                { 
+                {
                     Ukprn = ukprn,
+                    ContactName = parameters.ContactName,
```

Comment:
Your branch is not sync as I've seen these changes somewhere before?

Reply:
merged master -> 2037 -> 2059 and it should be in sync now